bcgov/moh-prime
# PRIME-1500 Create mock test data for local load testing (defect fix)

Updated to `develop` EnrolleeFactory and fixed defect related to Enrollee database sequence

A way to test:
1. Generate data using this command-line program
2. Confirm data works with Admin screens
3. Create enrollees and sites using web application
4. No errors should be encountered
